tiledb/sm/storage_manager/storage_manager.cc

```diff
@@ -2897,5 +2900,24 @@ Status StorageManager::set_default_tags() {
   return Status::Ok();
 }
 
+URI StorageManager::make_uri(const char* path, const Config* config) const {
+  if (!config) {
+    config = &config_;
+  }
+  std::string cpp_path =
+      (path == nullptr) ? std::string("") : std::string(path);
+  if (config == nullptr) {
+    return URI(cpp_path);
+  }
+
+  bool found = false;
+  std::string local_root_path = config->get("vfs.local_root_path", &found);
```

Comment:
? Should something somewhere, 'config' construction, and possibly setting of the value, verify validity of at least form of 'local_root_path'?  Or possibly the config->get() itself, tho that's potentially 'late' to report...